Scripts/Tracking/Follow/Modifier/SourceFollowActiveTarget.cs

```diff
@@ -18,50 +18,38 @@ public class SourceFollowActiveTarget : FollowModifier
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to modify.</param>
         /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method